yoti-sdk-spring-security/src/test/java/com/yoti/api/spring/security/web/YotiAuthenticationFilterTest.java

```diff
@@ -28,6 +24,9 @@
 
     private YotiAuthenticationFilter filter;
 
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
```

Comment:
What's the reason behind this change? What was wrong with `@RunWith` (seems to be the convention used by all other UTs -- at leas the random ones I checked)?

I would suggest to stick with the conventions. A change like this (where the same pattern is used elsewhere) should be done everywhere or not at all. Otherwise as times goes one the codebase becomes a myriad of different solutions to the same problem for no reason at all.